user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -6,7 +6,24 @@ uid: Configuring_external_authentication_via_an_identity_provider_using_SAML
 
 From DataMiner 9.6.11 onwards, it is possible to configure external authentication via an identity provider service using SAML (Security Assertion Markup Language).
 
-For this to be possible, a trust relationship must be established between the service provider (i.e. DataMiner) and the identity provider. This is done by exchanging SAML metadata files. The following metadata must be shared between the service provider (i.e. DataMiner) and the Identity Provider service: Entity ID, Cryptographic Keys, Protocol Endpoints (bindings, locations).
+## Requirements
+
+- DataMiner integrates with **identity providers using version 2.0 of the SAML protocol**. Compatibility with older SAML versions is not supported.
+
+- The identity provider (IDP) must support **Redirect binding** for communication between the service provider and the identity provider. Most SAML IDPs support Redirect binding by default.
+
+- Dataminer uses service provider initiated **Single Sign-On (SSO)** through Redirect binding. It does not support the use of POST or SOAP binding for requests. However, standard POST binding is used for responses.
+
+  > [!TIP]
+  > For a comprehensive understanding of the SAML process, including encoding and encryption guidelines that Dataminer follows, refer to the official SAML Documentation: [SAML Technical Overview](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0-cd-02.html#5.1.2.SP-Initiated%20SSO:%20%20Redirect/POST%20Bindings|outline).
+
+- From DataMiner 10.3.4/10.4.0 onwards, SAML assertions must be signed for proper integration. Non-signed SAML assertions will not be supported.
```

Comment:
I feel like we could delete this last item, @MichielMN. The page already explicitly states that, starting from DataMiner 10.3.4/10.4.0 onward, the "WantAssertionsSigned" flag is supported and must be configured as "true." This seems more like a step users should follow when consulting the actual guide rather than a distinct requirement.

Reply:
That's true, that's good for me